src/classes/HH_OppContactRoles_TEST.cls

```diff
@@ -504,5 +504,143 @@ private class HH_OppContactRoles_TEST {
             system.assertEquals(0, listSoft[4]);
         }
     }
-       
+
+    /*********************************************************************************************************
+    * @description Tests that an Opportunity Contact Role is created when the deceased member is set as the primary
+    * @return void
+    */
+    private static testMethod void opportunityDeceasedPrimaryAccount() {
+        
+        npe01__Contacts_and_Orgs_Settings__c contactSettingsForTests = UTIL_CustomSettingsFacade.getContactsSettingsForTests(
+            new npe01__Contacts_and_Orgs_Settings__c (
+                npe01__Opportunity_Contact_Role_Default_role__c = 'Donor'));
+        
+        npo02__Households_Settings__c householdSettingsForTests = UTIL_CustomSettingsFacade.getHouseholdsSettingsForTests(
+            new npo02__Households_Settings__c (
+                npo02__Household_Contact_Roles_On__c = true,
+                npo02__Household_Member_Contact_Role__c='Household Member'));
+
+        UTIL_UnitTestData_TEST.createAccountContactTestData(CAO_Constants.HH_ACCOUNT_TYPE, 1, 1, 1);
+        Account acct = UTIL_UnitTestData_TEST.listAccT[0];
+        Contact con = UTIL_UnitTestData_TEST.listConT[0];        
+        con.Deceased__c = true;  
+        update con;
+                
+        //create opportunity
+        Opportunity o = new Opportunity(
```

Comment:
This variable doesn't seem to have a very insightful name. Please give variables names that suggest their purpose.